src/ui/buttons/buttonDesigns/alternateSlideLogoDesign.jsx

```diff
@@ -0,0 +1,209 @@
+/* @flow */
+/** @jsx node */
+import { LOGO_CLASS } from '@paypal/sdk-logos/src';
+import { node, Fragment, type ChildType } from 'jsx-pragmatic/src';
+
+import { CLASS } from '../../../constants';
+import { BUTTON_SIZE_STYLE } from '../config';
+
+import type {  ContentOptions } from './types';
+
+type AnimationProps = {|
+    paypalLabelContainerElement : Object,
+    paypalLogoStartingLeftPosition : string,
+    designContainer : Object,
+    labelTextFontSize : number
+|};
+
+export const ALTERNATE_SLIDE_LOGO_CONFIG = {
```

Comment:
Isn't this a constant in one of your other PRs? If so can you use that here instead so that the design matches?

Reply:
yes, I am planing to create a constants file and export it, so we can re use it every where where we need it